tests/mpu_noexec_ram/Makefile.ci

```diff
@@ -0,0 +1,3 @@
+BOARD_INSUFFICIENT_MEMORY := \
+    stm32f030f4-demo \
+    #
```

Comment:
Not needed - even though the stm32f030f4 is tiny, this test still fits.

Reply:
Ah, good to know I copied the two Makefiles from `mpu_stack_guard`. Thanks for checking this! I just removed the `Makefile.ci`.